spec/calendar.html

```diff
@@ -459,8 +459,9 @@ <h1>ResolveISOMonth ( _fields_ )</h1>
         It returns the integer month.
       </p>
       <emu-alg>
-        1. Let _month_ be ? Get(_fields_, *"month"*).
-        1. Let _monthCode_ be ? Get(_fields_, *"monthCode"*).
+        1. Assert: _fields_ is created in PrepareTemporalFields, so the Get calls below can not fail.
```

Comment:
```suggestion
        1. Assert: _fields_ is an ordinary object with no more and no less than the own data properties listed in [Table 13](https://tc39.es/proposal-temporal/#table-temporal-field-requirements).
```
this seems clearer about why it can't throw, without needlessly coupling it to the name of its calling AO.